src/parser/parser.cpp

```diff
@@ -727,19 +725,12 @@ Command* Parser::nextCommand()
     }
   }
   Debug("parser") << "nextCommand() => " << cmd << std::endl;
-  if (cmd != NULL && dynamic_cast<SetOptionCommand*>(cmd) == NULL &&
-      dynamic_cast<QuitCommand*>(cmd) == NULL) {
-    // don't count set-option commands as to not get stuck in an infinite
```

Comment:
Are we sure that the problem mentioned in the comment is no longer an issue?

Reply:
The issue being referenced (I believe) is talking about a potential bad behavior of the resource manager itself. Thus, deleting this block of code makes the issue irrelevant.